contract-nft/dev.md

```diff
@@ -0,0 +1,95 @@
+<p align="center">
+  <a href="https://near.org/">
+    <img alt="NearMonotoneWhite" src="https://github.com/BalloonBox-Inc/near-oracle-contracts/blob/dev/images/monotone-black.png" width="550" />
+  </a>
+</p>
+
+
+# Minting NFTs on NEAR Protocol  :turtle: :fairy: :sunglasses:
+
+In this guide, you'll learn how to utilize the codebase of a Rust smart contract for NFT minting on NEAR Protocol. All it takes is 5 simple steps. Yet, before that you'll need to set up your environment. These instructions are for NEAR testnet.
+
+---
+
+
+### Configure CLI
+If you don't own a NEAR wallet yet, create one. Navigate to NEAR testnet [wallet](https://wallet.testnet.near.org) and click on 'Create Account'. Next, install the NEAR CLI.
+```bash
+npm install -g near-cli                                             # Install the NEAR CLI
+near                                                                # To see various possible commands run
+near login                                                          # Log into your NEAR testnet wallet
+near keys <youraccountname>.testnet                                 # Visualize your keys running
+```
+
+### Set up Rust
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh      # If you haven't installed Rust yet, install it now using rustup
+rustup target add wasm32-unknown-unknown                            # Add Wasm toolchain
+```
+
+Some -but not all- Smart Contracts (e.g., SCRT Network, Internet Computer, NEAR Protocol, etc.) compile to WebAssembly (Wasm) <br /> and that's why we add the :gear: toolchain for Rust.
+
+### Deploy
+
+Follow these 6 steps to deploy a Smart Contract on NEAR:
+1. :chains: **clone** the codebase
+2. :hammer_and_wrench: **compile** the contract
+3. :gear: **build** the contract
+4. :clapper: **deploy** the contract
+5. :airplane: **initialize** the contract
+6. :golf: **interact** with the contract
+
+Run in terminal,
+```bash
+git clone https://github.com/BalloonBox-Inc/near-oracle-contracts.git   <path_to_your_local_dir> # clone this Git Repo locally
+export A1=bruno.testnet                                                      # Export path to your testnet account name
+export A2=benji.testnet
+echo $A1      
+
+# now, ensure you are in the root folder and run
+yarn build && near deploy --wasmFile out/main.wasm --accountId $A1           # Deploy the contract
```

Comment:
Is there a compile step?

Reply:
Yes, the compile step is `yarn build`. You can also compile (locally) without building by running `./build.sh`, which will execute the commands in the associated shell file.